workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs

```diff
@@ -21,6 +21,8 @@ public abstract class WorkerBase : IDisposable
 
         public EntityGameObjectLinker EntityGameObjectLinker { get; private set; }
 
+        public bool AllowDynamicId = false;
```

Comment:
Rename from to match the flags' names?

Reply:
What do you think of this version, to replace `Allow` with `Use`: 

`WorkerBase.UseDynamicId` to match `useDynamicWorkerId`